keps/sig-auth/4872-harden-kubelet-cert-validation/kep.yaml

```diff
@@ -0,0 +1,40 @@
+title: Harden Kubelet Serving Certificate Validation in Kube-API server
+kep-number: 4872
+authors:
+  - "@g-gaston"
+owning-sig: sig-auth
+participating-sigs:
+status: implementable
+creation-date: 2024-09-24
+reviewers:
+  - TBD
+approvers:
+  - TBD
+
+see-also:
+replaces:
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "1.34"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "1.34"
+  beta: "1.35"
+  stable: "1.37"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: KubeletCertCNValidation
+    components:
+      - kube-apiserver
+disable-supported: true
+
+# The following PRR answers are required at beta release
+metrics:
```

Comment:
Nit, can you add here `kube_apiserver_validation_kubelet_cert_cn_total` and `apiserver_request_duration_seconds` which you referenced earlier in the document?